src/test/java/com/microsoft/graph/functional/OneDriveTests.java

```diff
@@ -77,5 +78,10 @@ public void failure(final ClientException ex) {
 		
     	chunkedUploadProvider.upload(callback);
     }
-    
+	@Test
+	public void testDownloadWithCustomRequest() throws IOException {
+		final String testDownloadFileId = "01RWFXFJG3UYRHE75RZVFYWKNUEBB53H7A";
+		final InputStream stream = testBase.graphClient.customRequest("/me/drive/items/"+testDownloadFileId+"/content", InputStream.class).buildRequest().get();
```

Comment:
We either need an explicit `close()` on the stream or [try-with-resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) block here.

Reply:
```suggestion
		try (final InputStream stream = testBase.graphClient.customRequest("/me/drive/items/"+testDownloadFileId+"/content", InputStream.class).buildRequest().get()) {
```